src/devices/schneider_electric.ts

```diff
@@ -2128,6 +2128,36 @@ const definitions: DefinitionWithExtend[] = [
             {vendor: 'Exxact', model: 'WDE003962'},
         ],
     },
+    {
+        zigbeeModel: ['NHMOTION/UNIDIM/1'],
+        model: 'NHMOTION/UNIDIM/1',
```

Comment:
I would expect a different model number is written on product page or device, can you use this?

Reply:
To be honest I'm not certain that this is actually a Schneider Electric white label product. It looks mostly identical to Elko EKO07250 and related devices, which is definitely produced by Schneider, but this device (EKO06984) seem like an earlier version.

In any case I haven't been able to find the Schneider product number anywhere.

I can change to `model: 'EKO06984'` and `vendor: 'Elko'`, but in that case I should perhaps move the definition over to `elko.ts`. In that case I would also have to duplicate a lot of the `schneiderElectricExtend` functionality.

Reply:
The `zigbeeModel` is different, `NHMOTION/UNIDIM/1` vs. `NHMOTION/DIMMER/1`.

Reply:
Then let's keep  `NHMOTION/UNIDIM/1`